docs/usage_guide/index.rst

```diff
@@ -0,0 +1,15 @@
+Usage Guide
+===========
+
+Here you can find Modin usage examples and tips of how you can further optimize the performance of your workload with Modin both for general use cases and for specific operations.
+
+.. toctree::
+    :maxdepth: 4
+
+    examples/index
+    optimization_guide/index
+    advanced_usage/index
```

Comment:
1. I do not see this page in readthedocs.
2. We should add partition API, setting configs to advanced_usage.

Reply:
https://readthedocs.org/projects/modin/builds/15737371/

Reply:
Added partitioning and configuring API sections to the user guide